src/qa4sm_reader/plotter.py

```diff
@@ -654,13 +656,20 @@ def barplot(
         fnames: list of file names with all the extensions
         """
         fnames, values = [], []
+        tc = False
+
+        # check if triple collocation results in Variables
+        if max([x[1].g for x in self._yield_values(metric=metric)]) > 2:
```

Comment:
Isn't this information (wheter TC is activated or not) already in the file attributes or in the qa4sm_reader.Image class attributes?

Reply:
yes in the qa4sm_reader.img triple is a list of tc metrics which is either empty of has a length. done.